### PR TITLE
Fix property of undefined name when creating a new space

### PR DESCRIPTION
**Pull request by samuveth**
image

### CODE REVIEW

File: src/views/SpaceSettings.vue [hunk 1]
```diff
@@ -272,7 +272,7 @@ watchEffect(async () => {
 });
 
 onMounted(() => {
-  setPageTitle('page.title.space.settings', { space: props.space.name });
+  setPageTitle('page.title.space.settings', { space: props.space?.name });
```
Comment (bonustrack) [inline src/views/SpaceSettings.vue:275]: With this change the title is then " Settings" if there is no space, but ideally it should be "Settings"

Comment (samuveth) [reply src/views/SpaceSettings.vue:275]:
```suggestion
  setPageTitle('page.title.space.settings', { space: props.space?.name ?? '' });
```
Would this change that?

Comment (samuveth) [reply src/views/SpaceSettings.vue:275]: Not sure how you can see the extra spacing

**Comment by samuveth** [comment]
https://github.com/snapshot-labs/snapshot/pull/1392/files

**Pull request closed without merging**